integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/inbound/transport/test/ESBJAVA4702JMSHeaderTest.java

```diff
@@ -73,7 +74,7 @@ public void JMSInboundEndpointHeaderTest()
             throws RemoteException, XMLStreamException, LogViewerLogViewerException, InterruptedException {
         logViewerClient = new LogViewerClient(contextUrls.getBackEndUrl(), getSessionCookie());
         logViewerClient.clearLogs();
-        axis2Client.sendRobust(getProxyServiceURLHttp("JMSProxy"), null, null, AXIOMUtil.stringToOM("<body/>"));
+        axis2Client.sendRobust(getProxyServiceURLHttp("jmsHeaderInboundEpTestProxy"), null, null, AXIOMUtil.stringToOM("<body/>"));
```

Comment:
Are we looking for a transport header here? We can use the wire monit for this

Reply:
No. we need to check if the inbound endpoint reads it properly. Checking at wire level is not the purpose.